tests/integration/mal_and_mea/test_model_run.py

```diff
@@ -22,3 +22,23 @@ def test_model_run(client, configured_project_with_label, rand_gen):
     assert annotation_group.model_run_id == model_run.uid
     assert annotation_group.data_row().uid == next(
         next(project.datasets()).data_rows()).uid
+
+
+def test_model_run_delete(client, model_run):
+    models_before = list(client.get_models())
+    model_before = models_before[0]
+    before = list(model_before.model_runs())
+
+    model_run = before[0]
+    model_run.delete_model_run()
+
+    models_after = list(client.get_models())
+    model_after = models_after[0]
+    after = list(model_after.model_runs())
+
+    assert len(before) == len(after) + 1
+
+
+def test_model_run_delete(client, model_run_annotation_groups):
+    # TODO
```

Comment:
I'm having issues running this test. Basically, it freezes when running and I suppose it's because of `model_run_annotation_groups` @msokoloff1

Reply:
Fixed by testing against staging env